pennylane/devices/default_qubit.py

```diff
@@ -412,9 +412,8 @@ def _apply_state_vector(self, state, device_wires):
         if not np.allclose(np.linalg.norm(state, ord=2), 1.0, atol=tolerance):
             raise ValueError("Sum of amplitudes-squared does not equal one.")
 
-        if (
-            len(device_wires) == self.num_wires
-            and sorted(device_wires.labels) == device_wires.labels
+        if len(device_wires) == self.num_wires and sorted(device_wires.labels) == list(
+            device_wires.labels
```

Comment:
Alternatively, you could do `device_wires.tolist()`, which has the same effect, but it a little more succinct.